src/main/java/ev3dev/actuators/lego/motors/BaseRegulatedMotor.java

```diff
@@ -186,7 +186,7 @@ public void coast() {
      * the motor to stop more quickly than coasting.
      */
     public void brake() {
-        this.setStringAttribute(STOP_COMMAND, BRAKE);
+        doStop(BRAKE, true);

```

Comment:
This is a ev3dev-addded function using the new common implemetation.

This will not work on BrickPi - it probably can´t physically short the motor pins, so it can´t stop the motor with the brake stop action.